README.md

```diff
@@ -51,11 +51,6 @@
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/226129009-9b72b795-ffdc-4008-95d0-655aa04059c4.png)

**[OPTIONAL]**

- [ ] Great work on implementing the changes requested by I saw an error in the console window :+1: Kindly fix it :+1: to make your application safe from breaking :+1: You are having props for your Rocket but you didn't pass any props from your parent component :fearful: Kindly fix it by passing the parameter from your parent component :+1: